be/src/olap/rowset/segment_v2/inverted_index_reader.h

```diff
@@ -72,6 +72,75 @@ class InvertedIndexIterator;
 class InvertedIndexQueryCacheHandle;
 class InvertedIndexFileReader;
 
+struct PrimitiveTypeConvertorHelper {
```

Comment:
It can be moved to more generic file like PrimitiveTypeTraits.

Reply:
I use a InvertedIndexQueryParam instead of this